symbolic-il2cpp/src/usym.rs

```diff
@@ -320,21 +342,33 @@ impl<'a> UsymSymbols<'a> {
         self.header.version
     }
 
-    fn get_string_from_offset(data: &[u8], offset: usize) -> Option<Cow<str>> {
+    /// Returns a string from the strings section at the given offset.
+    ///
+    /// Offsets are as provided by some [`UsymLiteHeader`] and [`UsymLiteLine`] fields.
+    fn get_string_bytes_from_offset(data: &'a [u8], offset: usize) -> Option<&'a [u8]> {
```

Comment:
For the time being I've exposed `_byte` equivalents for the string getters to make it easy to grab the string bytes to construct `FileInfo`s from. `FileInfo`s are strictly required for usym conversion - the managed file and its path require this.